tests/loader.ts

```diff
@@ -0,0 +1,16 @@
+intern.registerLoader((options) => {
```

Comment:
Providing our own loader lets us configure and call our shimAmdDependencies method rather than including all the configuration in the intern config. It probably makes sense to have this loader (or something like it) available for other packages to use.